pkg/cortex/cortex.go

```diff
@@ -113,6 +114,7 @@ type Config struct {
 	QueryRange       queryrange.Config               `yaml:"query_range"`
 	BlocksStorage    tsdb.BlocksStorageConfig        `yaml:"blocks_storage"`
 	Compactor        compactor.Config                `yaml:"compactor"`
+	ParquetConverter parquetconverter.Config         `yaml:"parquet_converter" doc:"hidden"`
```

Comment:
Do we usually mark a new component config as hidden?

Should we still add parquet storage feature as experimental in v1 guarantee?

Reply:
Make sense